packages/google_sign_in/lib/google_sign_in.dart

```diff
@@ -126,31 +124,63 @@ class GoogleSignIn {
     await _initialization;
     Map<String, dynamic> response = await _channel.invokeMethod(method);
     _currentUser = (response != null && response.isNotEmpty)
-        ? new GoogleSignInAccount._(this, response) : null;
+        ? new GoogleSignInAccount._(this, response)
+        : null;
     _streamController.add(_currentUser);
     return _currentUser;
   }
 
-  /// The currently signed in account, or null if the user is signed out
-  GoogleSignInAccount _currentUser;
+  /// Keeps track of the most recently scheduled method call.
+  Completer _lastMethodCompleter;
```

Comment:
I'd prefer to define a a super-small class with these two as final member variables. That makes it clearer that these two have to stay in sync.